tests/test_load_results.py

```diff
@@ -24,4 +24,4 @@ def test_load_results_mteb():
     model_results = mteb.load_results(models=[test_model_name])
     model_result = model_results[0]
     assert model_result.model_name == test_model_name
-    assert model_result.model_revision == "bf3bf13ab40c3157080a7ab344c831b9ad18b5eb"
```

Comment:
Can you provide script how you evaluated models? Because in mteb revision of `paraphrase-multilingual-MiniLM-L12-v2` is `bf3bf13ab40c3157080a7ab344c831b9ad18b5eb` and test shouldn't be changed
https://github.com/embeddings-benchmark/mteb/blob/8e6ee46408ca359833033b81aeec2be132cbfa0d/mteb/models/sentence_transformers_models.py#L86

Reply:
I ran this script in the README.md file in the mteb repository, under "Evaluating on a custom dataset":
```python
model = SentenceTransformer("model_name")
evaluation = MTEB(tasks=[MyCustomTask()])
evaluation.run(model)
```

Reply:
Also since you added benchmark, you can run it like 
```python
tasks = mteb.get_benchmark("NanoBEIR")
```